packages/embark/src/cmd/cmd_controller.js

```diff
@@ -504,58 +500,67 @@ class EmbarkController {
     const isSecondaryProcess = (engine) => { return engine.ipc.connected && engine.ipc.isClient(); };
 
     async.waterfall([
-      function initEngine(callback) {
+      callback => {
         engine.init({}, callback);
       },
-      function startServices(callback) {
+      callback => {
         let pluginList = engine.plugins.listPlugins();
         if (pluginList.length > 0) {
           engine.logger.info(__("loaded plugins") + ": " + pluginList.join(", "));
         }
 
-        engine.startService("web3");
-        engine.startService("deployment");
-        engine.startService("codeGenerator");
-        engine.startService("codeRunner");
-        engine.startService("console");
+        engine.registerModuleGroup("coreComponents");
+        engine.registerModuleGroup("stackComponents");
 
+        engine.registerModuleGroup("blockchain");
+        engine.registerModuleGroup("compiler");
+        engine.registerModuleGroup("contracts");
+        engine.registerModuleGroup("pipeline");
+        engine.registerModuleGroup("filewatcher");
+        engine.registerModuleGroup("storage");
+        engine.registerModuleGroup("communication");
+        engine.registerModulePackage('embark-deploy-tracker', {plugins: engine.plugins});
+        engine.registerModulePackage('embark-plugin-cmd', {embarkConfigFile: engine.embarkConfig, embarkConfig: engine.config.embarkConfig, packageFile: 'package.json'});
+        callback();
+      },
+      callback => {
         if (isSecondaryProcess(engine)) {
           return callback();
         }
```

Comment:
This ensure we aren't listening to file changes when console doesn't run in standalone mode